citi-onboarding/MeuApe-onepage
# Feature/intro

## What I did:

- create introduction

## How to test
1. Open the terminal and go to the folder of this project

~~~~typescript
git checkout develop
~~~~
~~~~typescript
git pull
~~~~
~~~~typescript
git checkout feature/intro
~~~~
~~~~typescript
git pull origin feature/intro
~~~~
~~~~typescript
yarn install
~~~~
~~~~typescript
yarn start
~~~~

2. In your browser go to <http://localhost:3000>

3. Check if the component looks just like the mockup in different screen sizes

4. Read the changed files section on GitHub

5. Test the things that you think are worth testing, even the ones that are not in this description
